CMakeLists.txt

```
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.22)
+cmake_minimum_required(VERSION 3.25)
```

```suggestion
cmake_minimum_required(VERSION 3.27)
```

Why not match the version specified in the Conan build requirements as you guarantee its support by making it a build requirement?

---

My bad, you right! This is a logical error. Thanks.